code-challenges.js

```diff
@@ -18,37 +44,107 @@ var temp3 = 212
 
 var myNumbers1 = [3, 7, 0, 6, -9]
 
+//create a function for the array
+const multBy5 = (array) =>{
+    //create a for loop for the array
+    for(let i = 0; i<array.length; i++){
+     //return and multiply each value at each index by 5. use *= to multiply each value at each index.
+    array[i] *= 5;
+    // return the array
+    }
+    return array;
+}
+//log results
+console.log(multBy5(myNumbers1));
 
 
-
-// --------------------3) Create a function that multiplies each number in the array by 5 using map.
-// Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
+// // --------------------3) Create a function that multiplies each number in the array by 5 using map.
+// // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
 
+//created a function with an array
+const multipleBy5 = (array) => {
+//used .map to multiple the array by 5 and return a new array
+    return array.map (value => value*5)
+}
 
+console.log(multipleBy5(myNumbers2));
 
 
 // --------------------4) Create a function that takes in an array and returns an array with only the values at odd indexes.
 // Use the test variables provided below. Expected output: [ "Chai tea", "Mocha", "Double espresso" ]
 
 var coffeeRun = ["Vanilla latte", "Chai tea", "Black coffee", "Mocha", "Americano", "Double espresso"]
 
+//declare a function that takes in an array
+const faveDrinks = (array) => {
+//return the array with the higher order function .filter()
+    return array.filter ((value, index) =>{
+//return the values of the array that have an odd index        
+        return index % 2 !== 0
+    })
+}
 
+//log the function
+console.log(faveDrinks(coffeeRun));
 
 
-// --------------------5) Create a function that removes all the vowels from a string.
-// Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
+// // --------------------5) Create a function that removes all the vowels from a string.
+// // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+//declare a function with an array
+const newString = (array) => {
+    //we need to convert the string into an array using split() so we can apply .filter() later on
+    let convertToArray = array.split("")
+    //we use higher order function .filter() to get rid of vowels
+    let noVowels = convertToArray.filter(value =>{
+        return value !== "a" && value !== "e" && value !== "i" && value !== "o" && value !== "u"
+    }
+    )
+    //convert the array back into a string and return the string
+    return noVowels.join("")       
+}
+
+console.log(newString(stringWithVowels1));
+console.log(newString(stringWithVowels2));
```

Comment:
Your solution totally works, but as I mentioned in the comment above, you just need to account for the uppercase vowels as well. 

Overall, good work with this problem!